versions/datastax/3.24.0/patch.patch

```diff
@@ -0,0 +1,156 @@
++                CCM_CLUSTER.populate(nodes, ipformat=ipformat)
+ 
+     try:
+         jvm_args = []
+ 
+         # This will enable the Mirroring query handler which will echo our custom payload k,v pairs back
+ 
+-        if 'graph' in workloads:
+-            jvm_args += ['-Xms1500M', '-Xmx1500M']
+-        else:
+-            if PROTOCOL_VERSION >= 4:
+-                jvm_args = [" -Dcassandra.custom_query_handler_class=org.apache.cassandra.cql3.CustomPayloadMirroringQueryHandler"]
++        # if 'graph' in workloads:
++        #     jvm_args += ['-Xms1500M', '-Xmx1500M']
++        # else:
++        #     if PROTOCOL_VERSION >= 4:
++        #         jvm_args = [" -Dcassandra.custom_query_handler_class=org.apache.cassandra.cql3.CustomPayloadMirroringQueryHandler"]
+         if len(workloads) > 0:
+             for node in CCM_CLUSTER.nodes.values():
+                 node.set_workloads(workloads)
+Index: tests/integration/standard/test_authentication_misconfiguration.py
+IDEA additional info:
+Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
+<+>UTF-8
+===================================================================
+diff --git a/tests/integration/standard/test_authentication_misconfiguration.py b/tests/integration/standard/test_authentication_misconfiguration.py
+--- a/tests/integration/standard/test_authentication_misconfiguration.py	(revision c88255f202a21bbbae35f16e603b0f10f2f2cf36)
++++ b/tests/integration/standard/test_authentication_misconfiguration.py	(date 1641374958430)
+@@ -19,6 +19,12 @@
+ 
+ class MisconfiguredAuthenticationTests(unittest.TestCase):
+     """ One node (not the contact point) has password auth. The rest of the nodes have no auth """
++    # TODO: 	Fix ccm to apply following options to scylla.yaml
++    # 	node3.set_configuration_options(values={
++    # 	'authenticator': 'PasswordAuthenticator',
++    # 	'authorizer': 'CassandraAuthorizer',
++    # 	})
++    # To make it working for scylla
+     @classmethod
+     def setUpClass(cls):
+         if not USE_CASS_EXTERNAL:
+Index: tests/integration/standard/test_prepared_statements.py
+IDEA additional info:
+Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
+<+>UTF-8
+===================================================================
+diff --git a/tests/integration/standard/test_prepared_statements.py b/tests/integration/standard/test_prepared_statements.py
+--- a/tests/integration/standard/test_prepared_statements.py	(revision c88255f202a21bbbae35f16e603b0f10f2f2cf36)
++++ b/tests/integration/standard/test_prepared_statements.py	(date 1641374958430)
+@@ -168,7 +168,7 @@
+     def _run_too_many_bind_values(self, session):
+         statement_to_prepare = """ INSERT INTO test3rf.test (v) VALUES  (?)"""
+          # logic needed work with changes in CASSANDRA-6237
+-        if self.cass_version[0] >= (3, 0, 0):
++        if self.cass_version[0] >= (2, 2, 8):
```

Comment:
What is it 2 2 8?

Reply:
revert it (also create a new pr in scylla python-driver rep)

Reply:
This change can cause the old Scylla versions will fail on this test.